src/main/java/org/generationcp/middleware/dao/LocationDAO.java

```diff
@@ -672,22 +459,6 @@ public long countByLocationAbbreviation(final String locationAbbreviation) {
 		}
 	}
 
-	public List<Location> getBreedingLocations() {
-		try {
-			final Criteria criteria = this.getSession().createCriteria(Location.class);
-			// set location types for Breeding Location
-			criteria.add(Restrictions.in(LocationDAO.LTYPE, Arrays.asList(Location.BREEDING_LOCATION_TYPE_IDS)));
-			criteria.addOrder(Order.asc(LNAME));
-
-			return criteria.list();
-		} catch (final HibernateException e) {
-			LocationDAO.LOG.error(e.getMessage(), e);
-			throw new MiddlewareQueryException(
-				this.getLogExceptionMessage("getBreedingLocationsByProgramUUID", "", null, e.getMessage(), "Location"), e);
-
-		}
-	}
-
 	@Deprecated
 	public List<LocationDetails> getFilteredLocations(final Integer countryId, final Integer locationType, final String locationName) {
```

Comment:
Why are there changes to LocationDAO and MethodDAO for this ticket?

Reply:
Because, I also made a cleanup in GermplasmDataManager.